playground/src/CodeEditor.tsx

```diff
@@ -2,21 +2,32 @@ import * as React from 'react';
 import * as monacoEditor from 'monaco-editor';
 import { FlexColDiv } from './FlexDivs';
 
-export interface ICommentSyntaxMarker {
-  message: string;
-
+export interface ISyntaxLocation {
   /**
-   * Beginning
+   * Beginning position as a character index of the text in the editor
    */
   pos: number;
 
   /**
-   * End
+   * End position as a character index of the text in the editor
    */
   end: number;
 }
 
-export interface IMonacoWrapperProps {
+export interface ISyntaxMarker extends ISyntaxLocation {
+  message: string;
+}
+
+export interface ISyntaxStyle extends ISyntaxLocation {
+  className: string;
+}
+
+const hashSymbol: unique symbol = Symbol('identifier');
```

Comment:
>hashSymbol [](start = 6, length = 10)

Symbols are a new and relatively poorly understood feature.  Could you add some code comments explaining how this symbol is used?  I was also unclear why the debug string is `identifier` but the variable name is `hashSymbol`. It's a little counterintuitive.

Reply:
Oops. I initially called it "identifier"

---
In reply to: [223964796](https://github.com/Microsoft/tsdoc/pull/93#discussion_r223964796) [](ancestors = 223964796)

Reply:
I refactored this out.

---
In reply to: [223970253](https://github.com/Microsoft/tsdoc/pull/93#discussion_r223970253) [](ancestors = 223970253,223964796)